vizro-core/src/vizro/models/_navigation/_navigation_utils.py

```diff
@@ -0,0 +1,21 @@
+import itertools
+
+from vizro.managers import model_manager
+
+
+def _validate_pages(pages):
+    """Validator for reuse in `Navigation` and `Accordion` to validate pages."""
+    from vizro.models import Page
+
+    pages_as_list = list(itertools.chain(*pages.values())) if isinstance(pages, dict) else pages
+
+    if not pages_as_list:
+        raise ValueError("Ensure this value has at least 1 item.")
+
+    # Ideally we would use dash.page_registry or maybe dashboard.pages here, but we only register pages in
+    # dashboard.pre_build and model manager cannot find a Dashboard at validation time.
+    # page[0] gives the page model ID.
```

Comment:
@antonymilne - I've checked to pass on the `dashboard.pages` here again, but as I correctly remembered, this doesn't seem to work as, at the point of validation, a model with the type Dashboard cannot be found in the model manager.

So you'll get something like below - any ideas?
![Screenshot 2023-10-17 at 18 14 39](https://github.com/mckinsey/vizro/assets/90609403/9575259a-eb31-40a4-84b2-bfa85ad862f4)

Reply:
Ahhh yes I remember... Thanks for checking!

It's not easy to fix here, so let's leave as is for now. And eventually the fix to #111 will improve this.